domains.yml

```diff
@@ -1,21 +1,28 @@
 - id: 'github.com'
-  description: 'github.com'
+  description: 'The leading software development platform: github.com'
+
   client-api: 'github'
-  url: 'https://api.github.com/repositories?since=129403500'
-  rawBaseUrl: 'https://raw.githubusercontent.com/'
+  apiOrgUrl: 'https://api.github.com/orgs/{{.OrgName}}/repos'
```

Comment:
client-api is snake case, apiOrgUrl is camel case, can we use only one standard? :-)

Reply:
in a config file I prefer the snake case version